.github/workflows/trunk.yml

```diff
@@ -273,10 +273,12 @@ jobs:
   test-llava-runner-macos:
     name: test-llava-runner-macos
     uses: pytorch/test-infra/.github/workflows/macos_job.yml@main
+    with:
+      python-version: "3.10"
```

Comment:
Default is 3.9: https://github.com/pytorch/test-infra/blob/main/.github/workflows/macos_job.yml#L75

Reply:
So here it says `python-version` should be a sub attribute of `workflow-call: inputs`

Reply:
Yeah, `with` is what actions uses to override the `inputs` in the included module. There's already a `with` below this which I missed that specifies python 3.11, but that didn't work the first time. Maybe you can't install python 3.11 on macos-14-xlarge for whatever reason (so it defaults to 3.9). I'll try a different mac model - https://github.com/pytorch/executorch/blob/main/.github/workflows/trunk.yml#L182